src/Core/StorageManager.php

```diff
@@ -105,35 +106,55 @@ public static function unregister($class)
 	/**
 	 * @brief Move resources to storage $dest
 	 *
+	 * Copy existing data to destination storage and delete from source.
+	 * This method cannot move to legacy in-table `data` field.
+	 *
 	 * @param string  $dest    Destination storage class name
 	 * @param array   $tables  Tables to look in for resources. Optional, defaults to ['photo']
 	 *
 	 * @retur int Number of moved resources
 	 */
-	public static function move($dest, $tables = null)
+	public static function move(string $dest, $tables = null)
```

Comment:
This type hint isn't available for PHP 5.6, please wait until version `2019.03`.